docs/wiki/Solution creation.md

```diff
@@ -405,8 +405,14 @@ on:
       - 'network-hub-rg/Parameters/**'
       - '.github/workflows/network-hub.yml'
 
+permissions:
+  id-token: write      # OIDC
+  contents: read       # OIDC
+  checks: write        # enricomi/publish-unit-test-result-action
+  pull-requests: write # enricomi/publish-unit-test-result-action
+
 env:
-  AZURE_CREDENTIALS: ${{ secrets.AZURE_CREDENTIALS }}
+  AZURE_CREDENTIALS: ${{ secrets.AZURE_CREDENTIALS }} # TODO: Update this to use OIDC
```

Comment:
Note: here is a `TODO`

Reply:
Yepp, this is deffo not ready for review. Think I just wanted initial thoughts on the changes. But lets discuss in the Thursday call. Don't want to doc yet until we agree on more aspects on this PR.